src/Aguacongas.TheIdServer.Duende/Localization-de.json

```diff
@@ -0,0 +1,2166 @@
+  },
+  {
+	"key": "Expires",
+	"value": "Läuft ab"
+  },
+  {
+	"key": "No session",
+	"value": "Keine Sitzung"
+  },
+  {
+	"key": "allowed identity token signing algorithms",
+	"value": "erlaubte Algorithmen zur Signierung von Identitätstoken"
+  },
+  {
+	"key": "Reset password",
+	"value": "Passwort zurücksetzen"
+  },
+  {
+	"key": "Reset your password.",
+	"value": "Setzen Sie Ihr Passwort zurück."
+  },
+  {
+	"key": "Reset",
+	"value": "Zurücksetzen"
+  },
+  {
+	"key": "Reset password confirmation",
+	"value": "Bestätigung des Passwortzurücksetzens"
+  },
+  {
+	"key": "Your password has been reset.",
+	"value": "Ihr Passwort wurde zurückgesetzt."
+  },
+  {
+	"key": "Please click here to log in",
+	"value": "Bitte klicken Sie hier, um sich anzumelden"
+  },
+  {
+	"key": "Register as a new user",
+	"value": "Als neuer Benutzer registrieren"
+  },
+  {
+	"key": "Resend email confirmation",
+	"value": "E-Mail-Bestätigung erneut senden"
+  },
+  {
+	"key": "Resend",
+	"value": "Erneut senden"
+  },
+  {
+	"key": "The machine account password is required.",
+	"value": "Das Maschinenkontokennwort ist erforderlich."
+  },
+  {
+	"key": "persist kerberos credentials",
+	"value": "kerberos-Anmeldeinformationen beibehalten"
+  },
+  {
+	"key": "persist ntlm credentials",
+	"value": "ntlm-Anmeldeinformationen beibehalten"
+  },
+  {
+	"key": "enable ldap",
+	"value": "ldap aktivieren"
+  },
+  {
+	"key": "domain",
+	"value": "Domäne"
+  },
+  {
+	"key": "machine account name",
+	"value": "Maschinenkontoname"
+  },
+  {
+	"key": "machine account password",
+	"value": "Maschinenkontokennwort"
+  },
+  {
+	"key": "enable ldap claim resolution",
+	"value": "ldap-Behauptungsauflösung aktivieren"
+  },
+  {
+	"key": "ignore nested groups",
+	"value": "verschachtelte Gruppen ignorieren"
+  },
+  {
+	"key": "claims cache absolute expiration",
+	"value": "Ablauf der absoluten Ansprüche-Cache"
+  },
+  {
+	"key": "claims cache sliding expiration",
+	"value": "Ablauf der gleitenden Ansprüche-Cache"
+  },
+  {
+	"key": "require resource indicator",
+	"value": "Ressourcenindikator erforderlich"
+  },
+  {
+	"key": "Are you sure you want to leave this page?",
+	"value": "Sind Sie sicher, dass Sie diese Seite verlassen möchten?"
+  },
+  {
+	"key": "clone",
+	"value": "klonen"
+  },
+  {
+	"key": "Clone of {0}",
+	"value": "Klon von {0}"
+  },
+  {
+	"key": "Scope of",
+	"value": "Umfang von"
+  },
+  {
+	"key": "Allowed to",
+	"value": "Erlaubt zu"
+  },
+  {
+	"key": "scope of",
+	"value": "Umfang von"
+  },
+  {
+	"key": "allowed to",
+	"value": "Erlaubt zu"
+  },
+  {
+	"key": "metadata",
+	"value": "Metadaten"
+  },
+  {
+	"key": "Cannot have more than one URI per kind.",
+	"value": "Kann nicht mehr als eine URI pro Art haben."
+  },
+  {
+	"key": "Either a metadata or redirect URI must be set.",
+	"value": "Entweder eine Metadaten- oder Redirect-URI muss festgelegt sein."
+  },
+  {
+	"key": "Either a metadata or redirect URI must be set.",
```

Comment:
can you update this key with : "Either a metadata URI or a X509Certificate secret must be set."

Reply:
Should I do the same in the -fr.json file?

Reply:
I did, thx